docs/sidebars.ts

```diff
@@ -567,6 +567,11 @@ module.exports = {
             }
           ]
         },
+//        {
```

Comment:
I recommend we keep this hidden until we have more content in the cookbook.

Reply:
Do you want to merge this PR and we can contribute sections in separate PRs and then uncomment this in a follow up PR after?
cc @levlaz

Reply:
> Do you want to merge this PR and we can contribute sections in separate PRs and then uncomment this in a follow up PR after? cc @levlaz

Yes, exactly